.travis_install.sh

```diff
@@ -20,8 +20,7 @@ export PATH=/home/travis/miniconda3/bin:$PATH
 conda update --yes conda
 popd
 
-conda create -n testenv --yes python=$TRAVIS_PYTHON_VERSION pip pytest hdf5 \
-      numpy=$NUMPY_VERSION scipy=$SCIPY_VERSION numba scikit-learn statsmodels
+conda create -n testenv --yes python=$TRAVIS_PYTHON_VERSION pip pytest
```

Comment:
Well, pip doesn't install them a second time but already sees the latest versions, doesn't it? Anyways, if numba installs fine with pip these days, there is no reason to install it with conda; so this is good.

Reply:
It looks like `conda` wasn't respecting the versioning in `requirements.txt`, and was installing `numba` `v0.39.0` for `python` `v3.5`. This made `pip` install a different version, but not remove the old one. I think the older `numba` was being imported, causing the errors.

Here're the relevant lines from the build for a2c0d2b:

```
numba-0.39.0         | 2.4 MB    | ##################################### | 100%  # Line 589
Collecting numba>=0.40.0 (from -r requirements.txt (line 13)) # Line 706
```

I tried specifying the `numba` version a bit higher for `conda`, but was getting conflicts in dependency resolution. I figured if everything could be installed by `pip`, I knew the `requirements.txt` would be followed.

Reply:
Got it!  Makes sense!